src/pybind/mgr/dashboard/rbd_ls.py

```diff
@@ -61,3 +64,28 @@ def _get(self):
                 pass
             result.append(stat)
         return result
+
+    def _format_bitmask(self, features):
+        count = 0
+        names = ""
+        features_dict = {
+                1<<0: "layering",
```

Comment:
Nit: prefer for you to use the defined constants from here [1]

[1] https://github.com/ceph/ceph/blob/master/src/pybind/rbd/rbd.pyx#L342